src/pybind/mgr/dashboard/services/nvmeof_client.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=unexpected-keyword-arg
```

Comment:
I saw that pylint2.6.0 mishandles line-level disables which was true for me because it failed when I did

```
MessageToDict(message, including_default_value_fields=True,  # pylint: disable=unexpected-keyword-arg
                                         preserving_proto_field_name=True)  # type: ignore
```

so had to keep it here. this is only a lint error so I am suspecting our pylinter is a bit older and its way past time to update that.